### ISSUE TITLE
Custom tag assignment isn't working

### ISSUE THREAD

**Issue by kion** [tkashkin/GameHub#268]
It's not possible to assign (or un-assign) a custom tag to a game once it's created and assigned for the first time. 

Once tag is created and assigned to a game for the first time, when I open any game details and try to assign that same tag to another game (by checking the tag checkbox), it instead filters the entire game list by that tag, i.e. does the filtering by tag instead of assigning tag, just like top menu filter icon/menu does. 

If you open game details dialog again, you'd notice that tag doesn't seem to be assigned to the game (i.e. checkbox is not checked) - neither the first one or any other. But if you try to filter by that tag, the first game you tagged it with does show up in results (even though - again - if you check its details, it doesn't have the checkbox checked), while all other games (to which tag was assigned) do not show up. 

So, to sum up: 

- once custom tag is assigned to a game, data model saves that assignment for that one game and then never saves it for any other games
- game details doesn't reflect the state of the custom tag assignment is either case (checkbox always unchecked
- checking custom tag in the game details dialog triggers game list filtering instead of assigning the tag

Also, as already reported by other ticket, it's quite frustrating that custom tag can't be deleted.

**Comment by tkashkin** [comment]
It should be fixed.

**Comment by kion** [comment]
Just tested the new CI build and it works like a charm now - thanks for the quick fix!
